lib/Controller/WizardController.php

```diff
@@ -63,7 +64,8 @@ public function disable() {
 	 * @return TemplateResponse
 	 */
 	public function show() {
-		$theming = \OC::$server->getThemingDefaults();
+		/** @var Defaults $theming */
+		$theming = \OC::$server->query(Defaults::class);
```

Comment:
This is basically only using the proper public interface instead of a private one from an app. (either returned `OCA\Theming\ThemingDefaults` or `OC_Defaults`).